### PR TITLE
Package fix

### PR DESCRIPTION
**Pull request by calvinmetcalf**

### CODE REVIEW

**Comment by calvinmetcalf** [comment]
license is actually MIT not BSD, put in the git repo and set to private as it's not meant to be put onto npm (at the moment)

File: package.json [hunk 1]
```diff
@@ -13,8 +13,8 @@
   },
   "repository": {
     "type": "git",
-    "url": ""
+    "url": "https://github.com/codeforboston/finda.git"
   },
-  "author": "https://github.com/codeforboston/finda.git",
```
Comment (paulswartz) [inline package.json:18]: Should `author` still be listed?

Comment (calvinmetcalf) [inline package.json:18]: it's optional, you can add yourself back in if you want

On Wed, Feb 26, 2014 at 1:24 PM, Paul Swartz notifications@github.comwrote:

> In package.json:
> 
> >    },
> > -  "author": "https://github.com/codeforboston/finda.git",
> 
> Should author still be listed?
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/codeforboston/finda/pull/53/files#r10091748
> .

## 

-Calvin W. Metcalf

Comment (paulswartz) [inline package.json:18]: Since we don't have a domain/e-mail to put in there as of yet, I'm fine with leaving it out.  Thanks!

**Pull request merged**